frontend/website/src/LegacyPage.re

```diff
@@ -103,11 +160,11 @@ module Footer = {
             target="_blank">
             ...children
           </a>
-          {last ?
-             <span className="dn" /> :
-             <span className="f6 silver">
-               {ReasonReact.string({js| · |js})}
-             </span>}
+          {last
+             ? <span className="dn" />
+             : <span className="f6 silver">
+                 {ReasonReact.string({js| · |js})}
+               </span>}
```

Comment:
Looks like that opam refmt version strikes again. I have an idea to help with this, will talk later